docs/config/index.md

```diff
@@ -121,3 +121,19 @@ export default defineConfig(({ mode }) => {
   }
 })
 ```
+
+## Debugging a config file on VScode
+
+With a default `--configLoader bundle`, Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder and a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.
```

Comment:
```suggestion
With the default `--configLoader bundle` behavior, Vite writes the generated temporary configuration file to the `node_modules/.vite-temp` folder and a file not found error will occur when setting breakpoint debugging in the Vite config file. To fix the issue, add the following configuration to `.vscode/settings.json`:
```

I reworded here a bit to remove "Vite 6" since the docs should already be about vite 6. I also updated the last sentence to show the settings file instead so they know where to place it.

Reply:
Thanks, this looks clearer.